tests/test_multidict.py

```diff
@@ -397,6 +415,23 @@ def test_or2(self, cls: Type[MutableMultiMapping[str]]) -> None:
 
         assert {"key", "key2"} == {"key2"} | d.keys()
 
+    def test_or_not_implemented(self, cls: Type[MutableMultiMapping[str]]) -> None:
```

Comment:
```suggestion
    def test_bitwise_or_not_implemented(self, cls: Type[MutableMultiMapping[str]]) -> None:
```

Reply:
Oh, so the old names are a bit inaccurate too. Something to be fixed separately.